tiledb/core.cc

```diff
@@ -593,6 +593,36 @@ class PyQuery {
     }
   }
 
+#if TILEDB_VERSION_MAJOR == 2 && TILEDB_VERSION_MINOR == 6
```

Comment:
`TILEDB_VERSION_MINOR >= 6`?

Reply:
The API is currently experimental so I wanted to pin it to 2.6 for now.